src/pages/AnimalNew.js

```diff
@@ -1,9 +1,84 @@
 import React, { Component } from 'react'
+import { Form, FormGroup, Label, Input, Button } from "reactstrap";
+import { Redirect } from "react-router-dom";
 
 export class AnimalNew extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: {
+        name: "",
+        age: "",
+        enjoys: "",
+        image: "",
+      },
+      submitted: false,
+    };
+  }
+  handleChange = (e) => {
+    // console.log(e.target.name);
+    // console.log(e.target.value);
```

Comment:
If you think you are going to need to come back to this you can keep it. I think it should be fine to delete.